Cabal/src/Distribution/Simple/Configure.hs

```diff
@@ -1028,10 +1028,17 @@ mkProgramDb cfg initialProgramDb = programDb
         . setProgramSearchPath searchpath
         $ initialProgramDb
     searchpath =
-      map
-        ProgramSearchPathDir
-        (fromNubList $ configProgramPathExtra cfg)
-        ++ getProgramSearchPath initialProgramDb
+      getProgramSearchPath initialProgramDb
+        ++ map
+          ProgramSearchPathDir
+          (fromNubList $ configProgramPathExtra cfg)
+
+-- Note. We try as much as possible to _prepend_ rather than postpend the extra-prog-path
```

Comment:
maybe postpend → append

Reply:
append doesn't specify to front or back

Reply:
I am not going to debate a native speaker, but “add (something) to the end of a written document.” ([def](https://dictionary.cambridge.org/dictionary/english/append)).

Reply:
Fun! I also don't like how artificial "postpend" sounds but on the other hand maybe extra clarity is justified here...

Reply:
FWIW I was surprised by "append doesn't specify"; I've always understood it as meaning "to the end of".